channels_points.go

```diff
@@ -17,6 +17,20 @@ type ChannelCustomRewardsParams struct {
 	ShouldRedemptionsSkipRequestQueue bool   `json:"should_redemptions_skip_request_queue"`
 }
 
+// DeleteCustomRewardsParams...
```

Comment:
Can we remove these comments? They don't add any value. I recently removed all these in a PR. See #103.